app/templates/components/submission-metadata-common.hbs

```diff
@@ -1,9 +1,23 @@
 <h3>Common</h3>
 <p>
-    Here are some instructions
+    Please provide additional information about your article/manuscript below:
 </p>
 <div class="container">
     <form>
+        <div class="form-group row">
+            <label for="volume" class="col-sm-2 col-form-label"> Volume</label>
+
+            <div class="col-sm-10">
+                <input type="text" id="volume" class="form-control" />
+            </div>
+        </div>
+      <div class="form-group row">
+          <label for="issue" class="col-sm-2 col-form-label">Issue</label>
+
+          <div class="col-sm-10">
+              <textarea id="issue" rows="4" class="form-control" />

```

Comment:
This should probably be an `<input type="text"`, as 'issue' is probably only two or three digits long